wire-ios-data-model/Source/MLS/OneOnOne/OneOnOneResolver.swift

```diff
@@ -71,12 +71,16 @@ public final class OneOnOneResolver: OneOnOneResolverInterface {
     public func resolveAllOneOnOneConversations(in context: NSManagedObjectContext) async throws {
         let usersIDs = try await fetchUserIdsWithOneOnOneConversation(in: context)
 
-        for userID in usersIDs {
-            do {
-                try await self.resolveOneOnOneConversation(with: userID, in: context)
-            } catch {
-                // skip conversation migration for this user
-                WireLogger.conversation.error("resolve 1-1 conversation with userID \(userID) failed!")
+        await withTaskGroup(of: Void.self) { group in
+            for userID in usersIDs {
+                group.addTask {
+                    do {
+                        try await self.resolveOneOnOneConversation(with: userID, in: context)
```

Comment:
This function throws errors. Should the error be handled somewhere else?

Reply:
Good point, but it should be skipped and continue. There was a discussion in the predecessor pull request:
https://github.com/wireapp/wire-ios/pull/974#discussion_r1490610879